src/extension.ts

```diff
@@ -34,23 +34,11 @@ export async function activate(context: vscode.ExtensionContext) {
 		selectionCommandHandler(chatGptExtensionConfig.get("promptPrefix.addTests")!);
 	});
 
-	const findProblems = vscode.commands.registerCommand("vscode-chatgpt.findProblems", () => {
-		selectionCommandHandler(chatGptExtensionConfig.get("promptPrefix.findProblems")!);
-	});
-
-	const optimize = vscode.commands.registerCommand("vscode-chatgpt.optimize", () => {
-		selectionCommandHandler(chatGptExtensionConfig.get("promptPrefix.optimize")!);
-	});
-
-	const explain = vscode.commands.registerCommand("vscode-chatgpt.explain", () => {
-		selectionCommandHandler(chatGptExtensionConfig.get("promptPrefix.explain")!);
-	});
-
 	const clear = vscode.commands.registerCommand("vscode-chatgpt.clearSession", () => {
 		context.globalState.update("chatgpt-session-token", null);
 	});
 
-	context.subscriptions.push(view, freeText, addTests, findProblems, optimize, explain, clear);
```

Comment:
@ayim you don't need to change the extension code to update your queries. Simply use the visual studio code settings page to override these commands. Go to your settings(UI) and type `chatgpt` you will see all available settings to override. Please check out this section on readme: https://github.com/gencay/vscode-chatgpt#settings